spec2vec/Spec2Vec.py

```diff
@@ -69,43 +100,44 @@ def __init__(self, model: Word2Vec, intensity_weighting_power: Union[float, int]
             Default is False.
         """
         self.model = model
+        self.n_decimals = self._get_word_decimals(self.model)
         self.intensity_weighting_power = intensity_weighting_power
         self.allowed_missing_percentage = allowed_missing_percentage
         self.vector_size = model.wv.vector_size
         self.disable_progress_bar = not progress_bar
 
-    def pair(self, reference: SpectrumDocument, query: SpectrumDocument) -> float:
+    def pair(self, reference: Union[SpectrumDocument, SpectrumType],
+             query: Union[SpectrumDocument, SpectrumType]) -> float:
         """Calculate the spec2vec similaritiy between a reference and a query.
 
         Parameters
         ----------
         reference:
-            Reference spectrum document.
+            Reference spectrum or spectrum document.
         query:
-            Query spectrum document.
+            Query spectrum or spectrum document.
 
         Returns
         -------
         spec2vec_similarity
             Spec2vec similarity score.
         """
-        reference_vector = calc_vector(self.model, reference, self.intensity_weighting_power,
-                                       self.allowed_missing_percentage)
-        query_vector = calc_vector(self.model, query, self.intensity_weighting_power,
-                                   self.allowed_missing_percentage)
+        reference_vector = self._calculate_embedding(reference)
+        query_vector = self._calculate_embedding(query)
 
         return cosine_similarity(reference_vector, query_vector)
 
-    def matrix(self, references: List[SpectrumDocument], queries: List[SpectrumDocument],
+    def matrix(self, references: Union[List[SpectrumDocument], List[SpectrumType]],
+               queries: Union[List[SpectrumDocument], List[SpectrumType]],
                is_symmetric: bool = False) -> numpy.ndarray:
         """Calculate the spec2vec similarities between all references and queries.
 
         Parameters
         ----------
         references:
-            Reference spectrum documents.
+            Reference spectrums or spectrum documents
```

Comment:
Use consistent punctuation
```suggestion
            Reference spectrums or spectrum documents.
```